.github/workflows/bosh-release-checks.yaml

```diff
@@ -28,12 +28,12 @@ jobs:
           make package-specs
       - name: Check if there is any change
         run: |
-            [ $(git status --porcelain | wc -l) -eq 0 ] || \
-            { \
-              git status;\
-              echo "::error::Specs are out of date, run \"make package-specs\" to update";\
-              exit 1;\
-            }
+          if [ "$(git status --porcelain | wc -l)" -ne 0 ]; then
+            git status
+            git diff
+            echo "::error::Specs are out of date, run \"make package-specs\" to update"
+            exit 1
+          fi
```

Comment:
`make package-specs` can behave differently on macOS with M3 processors. let's show the diff to make the life of these people easier.

Reply:
How did you fix resolve this issue in your PR? By manually applying the diff?

I was thinking that we could remove https://github.com/cloudfoundry/app-autoscaler-release/blob/main/.github/workflows/dependency-updates-post-processing.yaml#L11 - that way, the GitHub workflow would take care of fixing up the package specs for human PRs as well.

Reply:
Yes, unfortunately, I had to apply the diff manually 😢 .

Your idea is brilliant, I will give this a try and let it run also for "human PRs" by removing the condition

Reply:
Thanks for the hint!

I've implemented your feedback (running post-processing for all PRs and removing stale PR approvals) and tested it successfully. The following commit was created by the automation af50a31, because I deleted something from the spec intentionally with 6106c67.